packages/cli/package.json

```diff
@@ -14,8 +14,9 @@
     "preinstall": "node ./scripts/preinstall.js",
     "test": "jest --env node --verbose --runInBand --bail --forceExit",
     "test-unit": "yarn test test/unit/",
-    "test-integration-cli": "rimraf test/fixtures/integration && ava test/integration.js --serial --fail-fast --verbose",
+    "test-integration-cli": "rimraf test/fixtures/integration && ava test/integration.js --serial --fail-fast --verbose && yarn test:integration",
     "test-integration-dev": "yarn test test/dev/",
+    "test:integration": "jest --env node --verbose --bail test/integration/",
```

Comment:
We've discussed improving the tests in this repo for a while. I think now is as good a time as any. This change only affects the new tests and uses Jest (which is already configured for the unit tests). 

I'll extrapolate on this change in our slack channel, but in short, by using a test framework that supports test scoping with beforeEach/afterEach blocks allows the integration tests for this feature to be a lot cleaner.

Reply:
I'm curious why these couldn't just be unit tests

Reply:
I believe our reasoning was that they are invoking the CLI directly. While yes, they are all just using `vc build` instead of `vc deploy` or something. The fact that it was using the compiled CLI made us put them in the integration bucket. 

I guess we could rewrite them to call `build` directly instead of invoking through CLI

Reply:
The CI has been really flaky with the integration tests I added because of how long they take to run sometimes. 

If it is sufficient, I'm in favor of moving them to unit tests to be more efficient.